routes/drones.js

```diff
@@ -2,36 +2,70 @@ const express = require('express');
 
 // require the Drone model here
 
+const DroneModel = require('../models/Drone.model') 
+
 const router = express.Router();
 
 router.get('/drones', (req, res, next) => {
   // Iteration #2: List the drones
-  // ... your code here
+  DroneModel.find()
+    .then((drones) => {
+      console.log(drones)
+      res.render('drones/list.hbs', {drones})
+    })
 });
 
 router.get('/drones/create', (req, res, next) => {
   // Iteration #3: Add a new drone
-  // ... your code here
+  DroneModel.create(req.body)
```

Comment:
this mongoose promise is not needed here. This is handled in the post request below. This route is only for rendering the hbs with the create form